packages/app/src/elements/member-view.ts

```diff
@@ -15,13 +15,14 @@ import "./list";
 import "./toggle";
 import { customElement, property, query, state } from "lit/decorators.js";
 import { css, html, LitElement } from "lit";
+import { base64ToString } from "@padloc/core/src/encoding";
 
 @customElement("pl-member-view")
 export class MemberView extends Routing(StateMixin(LitElement)) {
     readonly routePattern = /^orgs\/([^\/]+)\/members(?:\/([^\/]+))?/;
 
     @property()
-    memberId: string;
+    email: string;
```

Comment:
Identifying members by email is safer, since the account id is not always guaranteed to be set.